src/util/networks.tsx

```diff
@@ -110,6 +110,8 @@ export const networkFormFieldToPayloadName: Record<
   parent: "parent",
   security_acls: "security.acls",
   vlan: "vlan",
+  security_acls_default_egress: "security.acls.default.egress.action",
+  security_acls_default_ingress: "security.acls.default.ingress.action",
```

Comment:
This is the correct way to have the keys persist on save. I also see the new fields in the API response after saving.

What is missing to read the changes is to add the two new fields also to the `toNetworkFormValues` function in `networkForm.tsx`.